news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,152 @@
+
+For the first round of losers' bracket matches, it would be ::{ flag=UA }:: Ukraine to pull off the only upset of the week, as far as pick'ems were concerned. After defeating ::{ flag=BR }:: Brazil 6–1 despite spectators favouring them 7:3, it turned out the country that doesn't care about the odds has an osu! World Cup team to match. They would go on to end their run against the ::{ flag=PH }:: Philippines, but not without showing us the undisputed match of the week — read on for a full analysis [further below](#philippines-vs.--ukraine).
+
+In the second losers' bracket round last weekend, there were again no pick'em surprises, but the winners all stood out for their own reasons:
+
+- ::{ flag=PL }:: Poland's 6–0 win against ::{ flag=TR }:: Türkiye completed their perfect 12–0 weekend, bouncing back from their singular point loss so far and securing a spot in the top 12, all despite having replaced over half of their 2023 roster.
+- ::{ flag=CL }:: Chile took down ::{ flag=SE }:: Sweden 6–1 and look to be rounding into form as they seek to live up to expectations as the #7 seed.
+- ::{ flag=CN }:: China, running back a core roster that continues to stand the test of time, swept ::{ flag=NO }:: Norway 6–0 in a surprising outcome that was truly closer than the scoreline indicated.
+
+After all is said and done, it is with great respect that we bid adieu to ::{ flag=TR }:: Türkiye for achieving their best world cup finish ever along with ::{ flag=NO }:: Norway, ::{ flag=SE }:: Sweden and ::{ flag=UA }:: Ukraine at 13th–16th, as well as ::{ flag=BR }:: Brazil, ::{ flag=FI }:: Finland, ::{ flag=FR }:: France, ::{ flag=ID }:: Indonesia, ::{ flag=JP }:: Japan, ::{ flag=RO }:: Romania, ::{ flag=TH }:: Thailand and ::{ flag=TW }:: Taiwan for finishing their 2024 campaigns in the top 24.
+
+### ::{ flag=PH }:: Philippines vs. ::{ flag=UA }:: Ukraine
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116222613)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2304248553)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cld2spAZQ-M?start=792&end=947" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The best match of the Quarterfinals may have been one you didn't even know about or watch. With three matches all happening at once, ::{ flag=PH }:: Philippines (seed #9) vs. ::{ flag=UA }:: Ukraine (seed #22), drew the short straw of not being streamed on the official Twitch channels. As it happened, this unofficial stream would include the only tiebreaker match of the weekend
```

Comment:
what unofficial stream? should it be hyperlinked or?
```suggestion
The best match of the Quarterfinals may have been one you didn't even know about or watch. With three matches all happening at once, ::{ flag=PH }:: Philippines (seed #9) vs. ::{ flag=UA }:: Ukraine (seed #22), drew the short straw of not being streamed on the official Twitch channels. As it happened, this unofficial stream would include the only tiebreaker match of the weekend.
```

Reply:
it's linked under the heading